routes.py

```diff
@@ -339,9 +345,13 @@ def createAccount():
     # print("Printing zip code")
     # print(zipcode)
     userExists = user.query.filter_by(username=wantedUsername).all()
-    # if userExists:
-    #     print("existing username try again")
-    #     return flask.jsonify({"message" : "Username is alreadt taken. Please try again with another username"})
+    if userExists:
```

Comment:
So I had commented this statement out because it was bugging out the onboarding page originally - is it working now that there is a front end implementation for this?

Reply:
Yes, this part is working now! If you'd like to double-check everything and run through the code feel free to